src/OptionList.tsx

```diff
@@ -183,16 +184,24 @@ const OptionList: React.RefForwardingComponent<
   // ========================= Keyboard =========================
   React.useImperativeHandle(ref, () => ({
     onKeyDown: (event) => {
-      const { which } = event;
+      const { which, ctrlKey } = event;
       switch (which) {
-        // >>> Arrow keys
+        // >>> Arrow keys & ctrl + n/p on Mac
+        case KeyCode.N:
+        case KeyCode.P:
         case KeyCode.UP:
         case KeyCode.DOWN: {
           let offset = 0;
           if (which === KeyCode.UP) {
             offset = -1;
           } else if (which === KeyCode.DOWN) {
             offset = 1;
+          } else if (isPlatformMac() && ctrlKey) {
```

Comment:
How about https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey

Reply:
Command + N would open a new Window, and this cannot be prevented.